pkg/registry/apis/folders/register_test.go

```diff
@@ -95,6 +95,24 @@ func TestFolderAPIBuilder_getAuthorizerFunc(t *testing.T) {
 				eval: "folders:create",
 			},
 		},
+		{
+			name: "user with delete permissions should be able to delete a folder",
```

Comment:
Would the reverse test (`user without delete permissions should not be able to delete a folder`) be valuable as well?

Reply:
yes, it would :) added